docs/authentication/social-connections/spotify.mdx

```diff
@@ -1,65 +1,72 @@
+  1. Navigate back to the **Spotify Developer Dashboard**. Select the **View client secret** link and copy the string.
+  1. Paste the string in **Client Secret** and select **Add connection**. Once the settings are saved, you should see a green **`Used for sign-in`** label next to Spotify in the **Provider** list.
 
-  ![The Spotify settings modal in the Clerk Dashboard. A red arrow is pointing to the 'Redirect URI' copy button.](/docs/images/authentication-providers/spotify/redirect-uri.webp)
+  ### Test your OAuth
 
-  ### Configure Spotify application
+  The simplest way to test your OAuth setup is by visiting your Clerk application's [Account Portal](/docs/customization/account-portal/overview), which is available out-of-the-box for all Clerk applications.
 
-  Navigate back to your Spotify app and paste the **Authorized redirect URI** you copied from the Clerk Dashboard into the **Redirect URIs** input.
-
-  Agree to Spotify's terms and conditions and click **Save**.
-
-  Once your Spotify app is created, you'll be redirected to the app dashboard. From there, navigate to **Settings**.
-
-  ![The Spotify Developer Dashboard for a specific application. A red arrow points to the 'Settings' button.](/docs/images/authentication-providers/spotify/settings.webp)
-
-  Copy the **Client ID** and **Client Secret**. Go back to the Clerk Dashboard, where the modal should still be open, and paste these values into the respective fields.
-
-  > [!NOTE]
-  > If the modal or page is not still open, navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the settings cog icon next to the Spotify option. You can now paste the **Client ID** and **Client Secret** into their respective fields.
-
-  ![The 'Basic information' tab of the Spotify application's settings. A red arrow points to the 'Client ID' and 'Client Secret'.](/docs/images/authentication-providers/spotify/copying-values-from-spotify.webp)
+  1. In the navigation sidebar of your **Clerk Dashboard**, select [**Account Portal**](https://dashboard.clerk.com/last-active?path=account-portal).
+  1. Next to the **Sign-in** URL, select **Visit**. The URL should resemble:
+     - **For development**: `https://your-domain.accounts.dev/sign-in`
+     - **For production**: `https://accounts.your-domain.com/sign-in`
+  1. On the sign-in page, you should see **Spotify** as an option. Use it to sign in with your Spotify account.
+</Steps>
 
-  ### Finished 🎉
+### Finished 🎉
 
-  Congratulations! Social connection with Spotify is now configured for your Clerk application.
-</Steps>
+Congratulations! Your users can now sign in and sign up for your Clerk application with their Spotify account.
```

Comment:
I've actually just been ending the guides with the "Test your OAuth" step, and not having this extra step here. what do you think about fully removing it?